wordpress.org/public_html/wp-content/plugins/theme-directory/class-wporg-themes-auto-review.php

```diff
@@ -0,0 +1,183 @@
+	 *
+	 * @see register_rest_route()
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<theme_slug>[^/]+)/(?P<ticket_id>[\d]+)/',
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => array( $this, 'update_item' ),
+				'permission_callback' => array( $this, 'update_item_permissions_check' ),
+			)
+		);
+	}
+
+	/**
+	 * A Permission Check callback which validates the request against a WP_ORG token.
+	 *
+	 * @param \WP_REST_Request $request The Rest API Request.
+	 * @return bool|\WP_Error True if the token exists, WP_Error upon failure.
+	 */
+	function update_item_permissions_check( $request ) {
+		return $this->permission_check_api_bearer( $request, 'AUTO_REVIEW_TRAC_API_GITHUB_BEARER_TOKEN' );
+	}
+
+	/**
+	 * A Permission Check callback which validates the a request against a given token.
+	 *
+	 * @param \WP_REST_Request $request  The Rest API Request.
+	 * @param string           $constant The constant that contains the expected bearer.
+	 * @return bool|\WP_Error True if the token exists, WP_Error upon failure.
+	 */
+	function permission_check_api_bearer( $request, $constant = false ) {
+		$authorization_header = $request->get_header( 'authorization' );
+		$authorization_header = trim( str_ireplace( 'bearer', '', $authorization_header ) );
+
+		if (
+			! $authorization_header ||
+			! $constant ||
+			! defined( $constant ) ||
+			! hash_equals( constant( $constant ), $authorization_header )
+		) {
+			return new \WP_Error(
+				'not_authorized',
+				__( 'Sorry! You cannot do that.', 'wporg-plugins' ),
+				array( 'status' => \WP_Http::UNAUTHORIZED )
+			);
+		}
+
+		return true;
+	}
+
+	/**
+	 * Returns whether the ticket number is for the correct theme.
+	 * 
+	 * @param string $keywords A string that should include the theme slug.
+	 * @param string $theme_slug The theme slug.
+	 * @return bool
+	 */
+	public function ticket_is_for_theme( $keywords, $theme_slug ) {
+		return ! empty( $keywords ) && strpos( $keywords , $theme_slug );
+	}
+
+	/**
+	 * Returns a cleaned up version of the content.
+	 * 
+	 * @param string $content GitHub specific language to format.
+	 * @return string
+	 */
+	public function get_formatted_content( $content ) {
+		$padding = '<br><br>';
+
+		// Add some space around the titles
+		$content = str_replace( "]:", "]:" . $padding, $content );
+		$content = str_replace( "[", $padding . "[", $content );
+
+		// Replace any links tp documentation
+		$content = preg_replace(
+			'%See: \b(([\w-]+://?|www[.])[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|/)))%s',
+			'See: <a href="$1">$1</a>',
+			$content
+		);
+
+		// Remove empty spaces
+		$content = trim( $content );
+
+		$disclaimer = "<p><i>The following has been generated by <a href='https://github.com/WordPress/theme-review-action'>Theme Review Action</a>, this is for informational purposes only.</i></p>";
```

Comment:
This could be a more explicit call-to-action (or call-to-inaction?). Perhaps add something like:

```This is intended to help find issues in the theme that fail to comply with the guidelines. It is not mandatory to pass all of these tests.
Theme developers: the documentation links may help you to discover and fix problems that might otherwise delay approval of your theme.
Theme reviewers: the test results may identify issues that need particular attention.
Errors and omissions in the test results can be reported to [GH issues link]
```